.github/workflows/tests.yml

```diff
@@ -49,7 +49,7 @@ jobs:
     # TODO: when actions supports using ${{env}} in job.*.if and not just job.*.steps.*.if, use this:
     #if: ${{ env.NIGHTLY }}
     # in the meantime, copy-paste this:
-    if: ${{ github.event_name == 'schedule' || github.event_name == 'workflow_dispatch' }}
+    # if: ${{ github.event_name == 'schedule' || github.event_name == 'workflow_dispatch' }}
```

Comment:
Is this commented out just so that the workflow runs for this PR? Will you revert before merging?